maat-court-data-api/src/main/java/gov/uk/courtdata/hearing/crowncourt/impl/CrownCourtProcessHelper.java

```diff
@@ -66,5 +66,18 @@ private boolean anyResultCodeMatch(final List<XLATResultEntity> resultCodes, fin
         return xlatResultRepository.fetchResultCodesForCCImprisonment();
     }
 
+    /**
+     * Any results code that has a work queue type 7 that means case is concluded.
+     * @param hearingResulted message
+     * @return true when case is concluded.
+     */
+    public boolean isCaseConcluded (final HearingResulted hearingResulted) {
+
+        List<XLATResultEntity> resultEntityList = xlatResultRepository.findByWqType(7);
```

Comment:
WQType enum already has the definition for conclusion queue. We should use it from there instead of hardcoding this 7.